openeogeotrellis/backend.py

```diff
@@ -1190,15 +1190,27 @@ def _set_permissions(job_dir: Path):
             with requests.get(model_id) as resp:
                 resp.raise_for_status()
                 metadata = resp.json()
-            if deep_get(metadata, "properties", "ml-model:architecture", default=None) is None:
-                raise OpenEOApiException(
-                    message=f"{model_id} does not specify a model architecture under properties.ml-model:architecture.",
-                    status_code=400)
-            checkpoints = []
-            assets = metadata.get('assets', {})
-            for asset in assets:
-                if "ml-model:checkpoint" in assets[asset].get('roles', []):
-                    checkpoints.append(assets[asset])
+            if "ml_model_metadata.json" in model_id:
```

Comment:
Maybe use endswith instead of in. I don't think this edge case will ever occur but better to be safe.